CHANGELOG-13.0.md

```diff
@@ -30,3 +30,197 @@ You can see all the changes done to package that you carry about with this tree.
+* [shopsys] removed deprecations before release 12.0 by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2768
+* [shopsys] introduced php-fpm base image by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2762
+* [project-base] reduced image url redis cache size by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2761
+* [shopsys] removed unused topMenu article placement by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2776
+* [shopsys] doctrine/orm 2.16.0 and higher are now marked as conflicting by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2691
+* [shopsys] upgraded doctrine/orm to latest version by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2775
+* [shopsys] updated overblog/graphql-bundle to stable version 1.0.0 with dependencies by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2787
+* [framework] added optional option to export data to Elasticsearch only for the specified domain by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2780
+* [project-base] removed unused code by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2777
+* [shopsys] changed default db server in adminer in local environment by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2803
+* [shopsys] graphql-bundle classes are now dumped so they can use composer autoload by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2830
+* [infrastructure] added information about GitHub limits and increased verbosity of composer install on CI servers by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2860
+* [framework] added optional manual readable frequency to crons.yaml by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2854
+
+### :book: Documentation
+* [shopsys] replaced Shopsys Framework with Shopsys Platform by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2682
+* [shopsys] updated open source license acknowledgements by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2709
+* [project-base] minor fix in Readme file by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2737
+* [docs] updated infrastructure schema by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2743
+* [docs] updated PHPStorm settings to use absolute paths for TypeScript by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2760
+* [docs] fixed build on readthedocs by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2811
+* [shopsys] added general upgrade notes to upgrade to Shopsys Platform by @grossmannmartin in https://github.com/shopsys/shopsys/pull/2817
+
+### :art: Design & appearance
+* [project-base] product videos UX fixes by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2746
+
+### :cloud: Infrastructure
+* [github-actions] updated GitHub actions for new storefront by @TomasLudvik in https://github.com/shopsys/shopsys/pull/2638
```

Comment:
I do not think that it is necessary to mention GitHub actions and Gitlab changes and deployment

Reply:
removed